contracts/EpochTokenLocker.sol

```diff
@@ -181,8 +181,15 @@ contract EpochTokenLocker {
     /** @dev used to determine how much time is left in a batch
       * return seconds remaining in current batch
       */
-    function getSecondsRemainingInBatch() public view returns(uint) {
-        return BATCH_TIME - (now % BATCH_TIME);
+    function getSecondsRemainingInBatch() public view returns(uint256) {
+        return BATCH_TIME - getBatchTime();
+    }
+
+    /** @dev used to determine number of seconds passed in current batch
+      * return seconds passed in current batch
+      */
+    function getBatchTime() public view returns(uint256) {
+        return now % BATCH_TIME;
     }
```

Comment:
Yup, that's about what I imagined

Reply:
As mentioned in the other comment, we could just have reused `getSecondsRemainingInBatch`

Reply:
Yes, I can still do that if this is what we prefer.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia0.giphy.com%2Fmedia%2FZ9EvIRmLEOS3JNFeVb%2Fgiphy.gif&token=lets_do_it" data-gitmeme-token="lets_do_it"><img src="https://media0.giphy.com/media/Z9EvIRmLEOS3JNFeVb/giphy.gif" title="Created by gitme.me with /lets_do_it"/></a>